test/test.js

```diff
@@ -132,3 +132,18 @@ sjcl.codec.hex = sjcl.codec.hex ||
     return sjcl.bitArray.clamp(out, len*4);
   }
 };
+
+// CCM ArrayBuffer implementation test relies on sjcl.codec.arrayBuffer.toBuffer
+sjcl.codec.arrayBuffer = sjcl.codec.arrayBuffer ||
```

Comment:
I guess this is a "hotfix" to make the tests work?
In test environments the code should always be built with all modules!